pkgs/development/python-modules/googlemaps/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
```

`isPy27` is not included by default.
```suggestion
{ stdenv, buildPythonPackage, fetchFromGitHub, isPy27
```

---

thanks i missed that